app/src/main/java/org/oppia/app/activity/HtmlParserTestActivty.kt

```diff
@@ -1,22 +1,26 @@
 package org.oppia.app.activity
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.text.Spannable
 import android.widget.TextView
 import org.oppia.app.R
 import org.oppia.util.parser.HtmlParser
+import org.oppia.util.parser.UrlImageParser
+import javax.inject.Inject
 
 /** This is a dummy activity to test Html parsing. */
-class HtmlParserTestActivty : AppCompatActivity() {
+class HtmlParserTestActivty : InjectableAppCompatActivity() {
```

Comment:
Typo in file name
`HtmlParserTestActivity`

Reply:
corrected.